.github/workflows/release.yaml

```diff
@@ -73,12 +73,13 @@ jobs:
           -DJPEGXL_ENABLE_VIEWERS=OFF \
           -DJPEGXL_ENABLE_PLUGINS=OFF \
           -DJPEGXL_ENABLE_OPENEXR=OFF \
+          -DJPEGXL_ENABLE_DEVTOOLS=ON \
 
     - name: Package release tarball
       run: |
         cd build
         tar -zcvf ${{ runner.workspace }}/release_file.tar.gz \
-          LICENSE* tools/{cjxl,djxl,benchmark_xl}
+          LICENSE* tools/{cjxl,djxl,benchmark_xl,cjpegli,djpegli,jxlinfo,butteraugli_main,ssimulacra2}
```

Comment:
Might some of the HDR tools (`pq_to_hlg`, `render_hlg`, `tone_map`, perhaps even `generate_lut_template` and `texture_to_cube`) make sense to add as well while we’re at it?

Reply:
Sure, feel free to add those. Going to merge this one like it is just to see if conformance is OK now.